wagtailio/static/sass/components/_sign-up-form.scss

```diff
@@ -71,6 +71,10 @@
         &::placeholder {
             color: $color--off-black;
         }
+
+        &:focus-visible {
+            outline: none;
+        }
```

Comment:
This removes the yellow outline, not specifically requested but was missed in review.

Reply:
isn't that an accessibility no-no?

Reply:
You're right, https://www.w3.org/WAI/WCAG22/Understanding/focus-visible.html. I should have known this one!